docs/development/FAQ.md

```diff
@@ -0,0 +1,10 @@
+# FAQ
```

Comment:
In general, this should have been in a separate commit. 

No need to spend time changing anything here, but just a note for future reference: to the best of your ability, try to have different PRs for different topics. This is a goal we strive for but it'll never be perfect.

Reply:
Thanks for pointing out! I will follow that for future tasks.